packages/eslint-plugin/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@redwoodjs/eslint-plugin",
```

Comment:
Make this should have a better name in case we have other eslint plugin packages? Or were you thinking that if we had other eslint plugins, they'd go in this package? I'm ok with that, just wanted to clarify. (If it's the latter, one nit: maybe `@redwoodjs/eslint-plugins`?)

Reply:
eslint has very specific rules for plugin package names. `@redwoodjs/eslint-plugins` is not a valid name. `@redwoodjs/eslint-plugin` is, and keep reading and it'll make sense 🙂   I've iterated on the design here a few times 🙂  The design I settled on was that we just have a single plugin. And then we add more rules to that one plugin instead. So now we just have a single `process-env-computed` rule. Adding another rule will give that rule a separate description, separate config etc

Reply:
Makes sense thanks!